src/GLideNUI-wtl/Settings.cpp

```diff
@@ -412,6 +414,7 @@ void saveCustomRomSettings(const char * _strIniFolder, const char * _strRomName)
 	WriteCustomSetting(video, windowedHeight);
 	WriteCustomSetting(video, fullscreenRefresh);
 	WriteCustomSetting(video, multisampling);
+	WriteCustomSetting(video, maxMultiSampling);
```

Comment:
I don't think that this is necessary. maxMultiSampling can't be per-game setting.